### ISSUE TITLE
Can't run /init.sh : no such file or directory

### ISSUE THREAD

**Issue by kiarn** [linuxmuster/linuxmuster-linbo7#7]
Hello,

I made an update of linbo today : 

```bash
$ dpkg -l | grep linbo
ii  linuxmuster-linbo-common7              2.4.3-3                                         all          linuxmuster-linbo common files: kernel, initrd and pxe boot configuration
ii  linuxmuster-linbo-gui7                 7.0.1-release                                   all          Linuxmuster Linbo GUI
ii  linuxmuster-linbo7                     2.4.3-3                                         all          linuxmuster-linbo scripts
```

The clients ( VM in Proxmox ) don't boot anymore, the linbo screen cannot be initialized, and I get repeatly : 

```
Can't run /init.sh : no such file or directory
Can't run /linbo.sh : no such file or directory
```

It's the same behaviour on 2 differents servers : did I miss something in the update ?

Thanks

Arnaud

**Comment by HappyBasher** [comment]
Hi Arnaud,

works for me. Perhaps linbofs is corrupted. Try `update-linbofs` on the server and force the clients to boot per pxe.
`cp /srv/linbo/boot/grub/grub.cfg /srv/linbo/boot/grub/grub.cfg.bak`
`cp /usr/share/linuxmuster/linbo/templates/grub.cfg.forced_netboot /srv/linbo/boot/grub/grub.cfg`
After booting the clients successfully restore grub.cfg.

Kind regards, Thomas

**Comment by kiarn** [comment]
Hi @HappyBasher 

Thanks, I already tried an `update-linbofs` yesterday, and force the netboot didn't solve the problem : 

![linbo](https://user-images.githubusercontent.com/10401079/122227418-2335d680-ceb7-11eb-80cf-78a2834caeda.png)

I made a downgrade to 2.4.3-2, and the problem does not appear anymore.
The client is a classic vm in proxmox with virtio network card and spice as display. 
How can I debug this to give you more usefull infos ?

Cheers

Arnaud

**Comment by HappyBasher** [comment]
Sorry, I published a corrupted package. There is now a rebuild with version 2.4.3-3.1 online.

**Comment by HappyBasher** [comment]
Note: This issue belongs to https://github.com/linuxmuster/linuxmuster-linbo.

**Comment by kiarn** [comment]
Thanks @HappyBasher I take note of the right repo.
Is that normal that the package `linuxmuster-linbo-common7` disappeared from the deb repo in testing ?
I cannot longer find/install `linuxmuster-linbo-common7=2.4.3-3`.

**Comment by HappyBasher** [comment]
Now it's complete. Forgot to copy over the second package (sic!).

**Comment by kiarn** [comment]
Thanks @HappyBasher.

Sorry to still annoy with this, but I'm now stuck in a linbo bootloop : I'm unable to start the os and come over and over again in the linbo screen.

I tried to format and partition the disk, but it sitll remains the same.
I didn't find anything in the logs, but I always get : 

```
error: no such device: cache.
error: no such device: /start.conf.
error: no such device: /linbofs64.lz.
```

I checked the label on the disk with `blkid` and the cache label was fine. The file `/start.conf` exists on the client, so I don't get why it complains. Any idea for this ?
Should I open anotehr issue for this ?

Thanks 

Arnaud
